Framework/include/MakeMVAVariables.h

```diff
@@ -114,7 +115,8 @@ class MakeMVAVariables
     }
 
 public:
-    MakeMVAVariables(const bool verb = false) : verb_(verb)
+    MakeMVAVariables(const bool verb = false) : verb_(verb) :
+        myVarSuffix_(myVarSuffix)
```

Comment:
Right now you are not passing in a "myVarSuffix". In fact this shouldn't compile.

Reply:
Right, this makes sense. As I said, I have no code that actually uses MakeMVAVariables, but yeah I can change that.